secatest/foundation_usage_v1_suite.go

```diff
@@ -739,15 +740,33 @@ func (suite *FoundationUsageV1TestSuite) TestSuite(t provider.T) {
 			},
 		},
 	}
-	suite.createOrUpdateInstanceV1Step("Create an instance", t, ctx, suite.regionalClient.ComputeV1, instance, nil, nil, secalib.CreatingResourceState)
+
+	expectInstanceMeta := secalib.NewRegionalWorkspaceResourceMetadata(instanceName,
+		secalib.ComputeProviderV1,
+		instanceResource,
+		secalib.ApiVersion1,
+		secalib.InstanceKind,
+		suite.tenant,
+		workspaceName,
+		suite.region)
```

Comment:
That is a lot of arguments, can't we create a builder like mechanism:

`expectInstanceMeta := secalib.Metadata().Tenant(suite.Tenant).Region(suite.Region).Kind(secalib.InstanceKind).Workspace(workspaceName).Name(instanceResource).Build()`